### PR TITLE
add(web): mock api

### PR DESCRIPTION
**Pull request by chdsbd**
This change adds mock endpoints that return static data. The web api and web ui are still very much work-in-progress.

### CODE REVIEW

File: web_ui/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```
Comment (sbdchd) [inline web_ui/.vscode/settings.json]: I'm thinking we have it so you open vscode in the root of the repo

thoughts?

File: web_ui/s/build [hunk 2]
```diff
@@ -0,0 +1 @@
+exec node scripts/build.js
```
Comment (sbdchd) [inline web_ui/s/build]: might want to put some shebangs on these scripts

File: web_ui/package.json [hunk 3]
```diff
@@ -93,11 +93,6 @@
     "webpack": "4.41.2",
     "webpack-dev-server": "3.9.0"
   },
-  "scripts": {
-    "start": "node scripts/start.js",
```
Comment (sbdchd) [inline web_ui/package.json]: should update the web_ui readme since they reference these

File: web_ui/.vscode/settings.json (hunk 1 above)
Comment (chdsbd) [reply web_ui/.vscode/settings.json]: I'm doubtful that will work with two python projects, no?

**Pull request merged**